lambdas/postgres-migration-async-operation/src/index.ts

```diff
@@ -8,20 +8,19 @@ const logger = new Logger({ sender: '@cumulus/postgres-migration-async-operation
 exports.handler = async () => {
   const stackName = process.env.stackName;
   const systemBucket = process.env.system_bucket;
-  const tableName = process.env.AsyncOperationsTable;
+  const dynamoTableName = process.env.AsyncOperationsTable;
 
   logger.info('Starting Postgres Migration Async Operation');
   const asyncOperation = await asyncOperations.startAsyncOperation({
     cluster: process.env.EcsCluster,
     lambdaName: process.env.MigrationLambda,
     asyncOperationTaskDefinition: process.env.AsyncOperationTaskDefinition,
     description: 'Data Migration 2 Lambda ECS Run',
-    operationType: 'Data Migration 2',
-    payload: {
-    },
+    operationType: 'Data Migration',
+    payload: {},
     stackName,
     systemBucket,
-    dynamoTableName: tableName,
+    dynamoTableName: dynamoTableName,
```

Comment:
This is unneeded, as the key is the same as the variable it references.   You should be able to just have `dynamoTableName`